app/src/main/java/org/fossasia/openevent/fragments/LocationsFragment.java

```diff
@@ -61,9 +59,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
             @Override
             public void onRefresh() {
                 DataDownload download = new DataDownload();
-                dbSingleton.clearDatabase(DbContract.Microlocation.TABLE_NAME);
```

Comment:
okay so where are you clearing the database. If you're not clearing it, we'll have both the new and the old values.